### PR TITLE
Make it api only

### LINKED ISSUE #90: Make an Api only branch
# Description
- Create an api only branch to be consumed by 'taverna-de-mordor-react'

#  Prerequisites
- change ally and auth to 'api' contract
- get every frontend based code deleted

# Additional Info
- make the pagination endpoints so our frontend consumes it properly with the lib[ react-table](https://tanstack.com/table/v8/docs/guide/installation)

### PR DESCRIPTION
**Pull request by Lucas-Marques-Leao**
# Related Issue
- #90 

# Proposed Changes
- every frontend resource was deleted except for adonis session and its configuration
- auth contract converted to "api" contract

# Additional Info
- This branch never pushes its info to the project's main branch, Dudu, don't kill me

### CODE REVIEW

**Review by Manuel-Antunes — CHANGES_REQUESTED** [review]
90 files ????? no i'll not review this

File: config/ally.ts [hunk 1]
```diff
@@ -19,15 +19,15 @@ import { AllyConfig } from '@ioc:Adonis/Addons/Ally'
...
+	|--------------------------------------------------------------------------
+	| Google driver
+	|--------------------------------------------------------------------------
+	*/
   google: {
     driver: 'google',
     clientId: Env.get('GOOGLE_CLIENT_ID'),
     clientSecret: Env.get('GOOGLE_CLIENT_SECRET'),
-    callbackUrl: Env.get('CLIENT_URL') + '/auth/google',
+    callbackUrl: 'http://localhost:3000/auth/google',
```
Comment (duardoqueiroz) [inline config/ally.ts]: change to environment

**Review by duardoqueiroz — COMMENTED** [review]
Ok! Repproved :)

File: app/Controllers/Http/EventsController.ts [hunk 2]
```diff
@@ -5,40 +5,38 @@ import UpdateValidator from 'App/Validators/Event/UpdateValidator'
...
       month: parseInt(month, 10),
-    })
+    }
   }
 
-  public async create({ view, bouncer }: HttpContextContract) {
+  public async create({ bouncer }: HttpContextContract) {
     await bouncer.with('EventPolicy').authorize('invoke')
-    return view.render('events/create')
   }
```
Comment (duardoqueiroz) [inline app/Controllers/Http/EventsController.ts:20]: Remove this method

File: app/Controllers/Http/EventsController.ts [hunk 3]
```diff
@@ -5,40 +5,38 @@ import UpdateValidator from 'App/Validators/Event/UpdateValidator'
...
-  public async edit({ response, view, params, bouncer }: HttpContextContract) {
+  public async edit({ response, params, bouncer }: HttpContextContract) {
     await bouncer.with('EventPolicy').authorize('invoke')
     const event = await Event.find(params.id)
     if (!event) {
       return response.notFound('Event Not Found')
     }
 
-    return view.render('events/edit', { event })
+    return event
```
Comment (duardoqueiroz) [inline app/Controllers/Http/EventsController.ts:36]: remove this method

File: docker-compose.yml [hunk 4]
```diff
@@ -9,3 +9,10 @@ services:
       POSTGRES_USER: ${PG_USER}
     ports:
       - 5432:${PG_PORT}
+
+  mongo:
+    image: mongo
+    ports:
+      - '27017:27017'
```
Comment (duardoqueiroz) [inline docker-compose.yml:16]: please, create environment to mongodb port

**Review by duardoqueiroz — CHANGES_REQUESTED** [review]
Please, solve my suggestions.

- Remove 'create' and 'edit' methods in crontrollers.
- Solve conflicts

**Pull request closed without merging**